runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -3191,11 +3188,10 @@ J9::Z::PrivateLinkage::addFECustomizedReturnRegDependency(int64_t killMask, TR::
       }
    return killMask;
    }
-
-////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
 // J9::Z::PrivateLinkage::buildDirectDispatch - build direct function call
 //   eg. Static, helpers... etc.
-////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
 TR::Register *
```

Comment:
It looks like a blank line was deleted and some slashes were removed and added accidentally.  It should look like this:

```
   }

////////////////////////////////////////////////////////////////////////////////
// J9::Z::PrivateLinkage::buildDirectDispatch - build direct function call
//   eg. Static, helpers... etc.
////////////////////////////////////////////////////////////////////////////////
TR::Register *
```